airbyte-integrations/connectors/source-salesforce/source_salesforce/spec.yaml

```diff
@@ -81,8 +88,7 @@ connectionSpecification:
             title: Search value
             order: 2
       title: Filter Salesforce Objects
-      description: >-
-        Filter streams relevant to you
+      description: Add filters to select only required stream based on `SObject` name. See the docs for more information.
```

Comment:
This description tell me what this does, but not why I would use it. Why would I want to filter only to the required stream based on the `SObject` name? Is it for performance reasons or something?